describe('throw', () => {
  it('should throw an error', () => {
    throw new Error('Failing test that should show up in the summary')
  })
})
